viadot/sources/s3.py

```diff
@@ -1,266 +1,132 @@
-import os
-from typing import Any, Dict, List
+from typing import List
+import boto3
 
 import pandas as pd
+import awswrangler as wr
 
-from ..config import get_source_credentials
-from ..exceptions import CredentialError
-from .base import Source
-from ..utils import add_viadot_metadata_columns
 
-
-class S3(Source):
+class S3:
```

Comment:
Every source should inherit from the `Source` base class.

Reply:
```suggestion
class S3(Source):
```